### DISCUSSION TITLE
[Question] I wonder how to set the gain of damping and stiffness.

### DISCUSSION THREAD

**Discussion by Holiclife-KTH** [isaac-sim/IsaacLab#1087] [Q&A]
Hi!
I have a question regarding the damping and stiffness parameters in the asset config file.
I created a new robot asset config file in the lab_assets folder to use the UR5e and 2F85 gripper. However, there is no reference on how to set the stiffness and damping parameters. Could you provide some tips or guidance on how to configure these parameters if there is a specific method?

![image](https://github.com/user-attachments/assets/2a0d7bf0-6a05-4cbb-ad2d-8f295312d798)

**Comment by Dhoeller19** [comment]
Hi @Holiclife-KTH,
There are two ways of setting the actuator gain. 
The first one is setting it in the robot config as shown in your screenshot. You can modify stiffness and damping in the actuators dict.
At runtime, you can modify them on the fly via an event, see https://github.com/isaac-sim/IsaacLab/blob/main/source/extensions/omni.isaac.lab/omni/isaac/lab/envs/mdp/events.py#L246-L297

**Comment by Holiclife-KTH** [comment]
Thanks for your response, @Dhoeller19.
However, I’m still curious about how to tune the stiffness and damping parameters.
For example, the stiffness and damping values for the Franka robot are 80 and 4, while the Universal Robot has values of 800 and 40.
Is trial and error (running and checking) the only method for tuning these parameters, or is there a more systematic approach?
